jobserver/management/commands/get_all_projects.py

```diff
@@ -0,0 +1,72 @@
+import csv
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from furl import furl
+
+from jobserver.models import Job, Project
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        """
```

Comment:
This is docstring is nearly identical to the other, possibly just needs trimming to the first line?

Reply:
certainly not a copypasta fail…